controllers/config/constants.go

```diff
@@ -55,4 +55,5 @@ const (
 	Plugins                   = "/plugins"
 	CSIKubeletRootDirPath     = "/var/lib/kubelet"
 	CSIScc                    = "spectrum-scale-csiaccess"
+	CacertMountPath           = "/var/lib/ibm/ssl/public/"
```

Comment:
should be either `CACertMountPath` or `caCertMountPath`. You may refer this for naming variables with initialisms https://github.com/golang/go/wiki/CodeReviewComments#initialisms